cosmo_tester/framework/examples/on_manager.py

```diff
@@ -41,13 +41,17 @@ def upload_blueprint(self):
             self.manager.client.blueprints.upload(
                 self.blueprint_file, self.blueprint_id)
 
-    def check_files(self):
+    def check_files(self, path=None, expected_content=None):
         instances = self.manager.client.node_instances.list(
             deployment_id=self.deployment_id,
             _include=['id', 'node_id'],
         )
 
-        expected_content = self.inputs['content']
+        if path is None:
+            path = self.inputs['path']
+        if expected_content is None:
+            expected_content = self.inputs['content']
+
         for instance in instances:
             if instance.node_id == 'file':
                 file_path = self.inputs['path'] + '_' + instance.id
```

Comment:
@geokala you are still using `self.inputs['path']` should be change to `path`

Reply:
You ruin all my best bugs. Just testing a fix for that.

Reply:
:P :P